docs/src/tool/index.md

```diff
@@ -33,9 +33,9 @@ Reach has a few sub-commands, each with their own options.
 
 However, all commands support the following options:
 
-+ The environment variable `REACH_VERSION` signifies what version of Reach to use.
++ The environment variable `{!cmd} REACH_VERSION` signifies what version of Reach to use.
```

Comment:
```suggestion
+ The environment variable `{!cmd} REACH_VERSION` tells the `reach` script what version of Reach to use.
```

I think specificity and clueing in readers to how Reach works currently would be helpful. @mattaudesse might have additional thoughts.

Reply:
That's true. @Craig-Elre we can ignore this suggestion.